ogre2/src/Ogre2ThermalCamera.cc

```diff
@@ -319,13 +347,18 @@ void Ogre2ThermalCameraMaterialSwitcher::preRenderTargetUpdate(
           if (minTemperature && maxTemperature)
           {
             // make sure the temperature range is between [0, 655.35] kelvin
+            // for 16 bit format and 10mK resolution
```

Comment:
```suggestion
            // make sure the temperature range is between [min,max]
            // for the given pixel format and camera resolution
```
I think it's better to take out the explicit reference to 16 bit format and 10mK resolution, now that these parameters are configurable. That way, users aren't confused by the comment in case the resolution and pixel format are different.

Reply:
43c80a9